packages/features/ee/workflows/lib/constants.ts

```diff
@@ -14,11 +14,32 @@ export const WORKFLOW_ACTIONS = [
   WorkflowActions.EMAIL_ADDRESS,
   WorkflowActions.SMS_ATTENDEE,
   WorkflowActions.SMS_NUMBER,
+  WorkflowActions.WHATSAPP_ATTENDEE,
+  WorkflowActions.WHATSAPP_NUMBER,
 ] as const;
 
 export const TIME_UNIT = [TimeUnit.DAY, TimeUnit.HOUR, TimeUnit.MINUTE] as const;
 
-export const WORKFLOW_TEMPLATES = [WorkflowTemplates.CUSTOM, WorkflowTemplates.REMINDER] as const;
+export const WORKFLOW_TEMPLATES = [
+  WorkflowTemplates.CUSTOM,
+  WorkflowTemplates.REMINDER,
+  WorkflowTemplates.REMINDER,
+  WorkflowTemplates.CANCELLED,
+  WorkflowTemplates.COMPLETED,
+  WorkflowTemplates.RESCHEDULED
+] as const;
+
+export const BASIC_WORKFLOW_TEMPLATES = [
+  WorkflowTemplates.CUSTOM,
+  WorkflowTemplates.REMINDER,
+] as const;
+
+export const WHATSAPP_WORKFLOW_TEMPLATES = [
```

Comment:
We can also add these new templates for SMS (but we can do that in a follow-up PR)

Reply:
I'll be happy to do it in a follow-up PR. This one feels too big already.